classes/helper/HelperCalendar.php

```diff
@@ -169,10 +169,10 @@ public function getDateFrom()
         return $this->_date_from;
     }
 
-    public function setDateTo($value = '')
+    public function setDateTo($value)
```

Comment:
```suggestion
    public function setDateTo($value = '')
```
That was initial value